addons/payment_transfer/models/payment_acquirer.py

```diff
@@ -16,21 +16,30 @@ class TransferPaymentAcquirer(models.Model):
     provider = fields.Selection(selection_add=[
         ('transfer', 'Manual Payment')
     ], default='transfer', ondelete={'transfer': 'set default'})
+    qr_code = fields.Boolean(
+        'Enable QR Codes', help="Enable the use of QR-codes for payments made on this provider.")
 
     @api.model
-    def _create_missing_journal_for_acquirers(self, company=None):
-        # By default, the wire transfer method uses the default Bank journal.
+    def _create_missing_journals(self, company=None):
+        """ Override of payment to assign the default Bank journal instead or Electronic. """
+        # Search for transfer acquirers having no journal
         company = company or self.env.company
-        acquirers = self.env['payment.acquirer'].search(
-            [('provider', '=', 'transfer'), ('journal_id', '=', False), ('company_id', '=', company.id)])
+        transfer_acquirers = self.env['payment.acquirer'].search([
+            ('provider', '=', 'transfer'),
+            ('journal_id', '=', False),
+            ('company_id', '=', company.id)
+        ])
 
+        # Pick the Bank journal
         bank_journal = self.env['account.journal'].search(
             [('type', '=', 'bank'), ('company_id', '=', company.id)], limit=1)
         if bank_journal:
-            acquirers.write({'journal_id': bank_journal.id})
-        return super(TransferPaymentAcquirer, self)._create_missing_journal_for_acquirers(company=company)
+            transfer_acquirers.write({'journal_id': bank_journal.id})
 
-    def transfer_get_form_action_url(self):
+        return super()._create_missing_journals(company=company)
+
+    @api.model
+    def _transfer_get_redirect_action_url(self):
         return '/payment/transfer/feedback'
```

Comment:
```suggestion
        return TransferController._accept_url
```
Actually, this just could be done from '_get_specific_rendering_values` directly.